app/connectors/Cache.scala

```diff
@@ -44,4 +47,6 @@ trait Cache {
   def fetch(cacheId: String)(implicit hc: HeaderCarrier): Future[Option[CacheMap]]
 
   def getEntry[A](cacheId: String, key: String)(implicit hc: HeaderCarrier, fmt: Format[A]): Future[Option[A]]
+
+  def remove()(implicit hc: HeaderCarrier)
```

Comment:
This you'll need to have the return type which will be `Future[HttpResponse]`. For now it's treated like Unit which is not the same with coparison with `httpCacheClient`